src/Form/index.tsx

```diff
@@ -913,16 +925,16 @@ function Form({
     // and would require a custom submit prior to payment setup (which is also disabled for draft).
     if (!_draft)
       for (let i = 0; i < activeStep.servar_fields.length; i++) {
-        const servar = activeStep.servar_fields[i].servar;
-        if (servar.type === 'payment_method') {
+        const field = activeStep.servar_fields[i];
+        if (field.servar.type === 'payment_method') {
           const integrationData = integrations?.stripe;
           const actionData: ActionData = {
-            servar,
+            servar: field,
```

Comment:
why is a field being passed as the servar? if we need a field, then we should just call it a field